x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecuritySystemIndices.java

```diff
@@ -756,6 +756,10 @@ private static XContentBuilder getTokenIndexMappings() {
                             builder.field("type", "boolean");
                             builder.endObject();
 
+                            builder.startObject("token");
+                            builder.field("type", "keyword");
+                            builder.endObject();
+
```

Comment:
@slobodanadamovic As far as I can tell, at least till v8.10, this change should by itself be enough to update the mapping of the tokens index automatically (the code is hard to follow, and I'm not sure what the plans post 8.10 are, so I would appreciate if you could double check it, please).

Reply:
Here's the work tracking how it should work post 8.10: https://github.com/elastic/elasticsearch/pull/97529

Reply:
Thanks for checking it and linking the PR. 👍  I've only had brief information that there was a plan to migrate system indices away from version. Hence why I mentioned it. But seems like we are still depending on it and the mapping update will be handled automatically.